packages/reporter/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@lage-run/reporter",
```

Comment:
minor nit suggestion, maybe this should be called `reporters` plural . it contains all the reporters :D